cves/kernel/CVE-2016-6130.yml

```diff
@@ -320,9 +316,9 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
-stacktrace:
+  answer: true
+  note: Reported-by Pengfei Wang <wpengfeinudt@gmail.com> Reviewed-by Michael Holzheu <holzheu@linux.vnet.ibm.com>\nSigned-off-by Martin Schwidefsky <schwidefsky@de.ibm.com> These are all found in the fix commit.
```

Comment:
This could use a little tidying up. Fix the formatting and remove the newlines.

Reply:
Thanks! Went through and fixed it.